metabase/metabase
# Migrate bg-brand-hover to JS colors

Epic https://github.com/metabase/metabase/issues/23954

How to test:
- There is a list of all affected places. Make sure they look the same in this branch and `master`.

1. Scalar
<img width="382" alt="Screenshot 2022-07-29 at 15 44 15" src="https://user-images.githubusercontent.com/8542534/181763302-a78c910d-ac3a-4d6c-bf1e-85a83d8ea583.png">

2. Pivot table - row toggle icon
<img width="596" alt="Screenshot 2022-07-29 at 15 44 03" src="https://user-images.githubusercontent.com/8542534/181763357-cbf58fce-d5bd-4d62-8470-e15a977fb085.png">

3. Table - column resize handler
<img width="1097" alt="Screenshot 2022-07-29 at 15 46 26" src="https://user-images.githubusercontent.com/8542534/181763669-1246d2ba-814a-42a0-96f7-68209cd7f24d.png">

4. Table - buttons in tooltips
<img width="315" alt="Screenshot 2022-07-29 at 15 48 41" src="https://user-images.githubusercontent.com/8542534/181764033-061b31a3-faf6-4ec7-93ce-a50e7f9d40b5.png">

5. Subscriptions - list of existing subscriptions
<img width="395" alt="Screenshot 2022-07-29 at 15 52 45" src="https://user-images.githubusercontent.com/8542534/181764609-d27fec58-bf34-4373-9023-02084ee3f8ed.png">

6. Subscriptions - creating new subscription
<img width="392" alt="Screenshot 2022-07-29 at 15 53 38" src="https://user-images.githubusercontent.com/8542534/181764764-337674e8-f894-4372-b40b-b7079a4f6953.png">

7. Query builder - buttons for visualization types
<img width="648" alt="Screenshot 2022-07-29 at 15 55 14" src="https://user-images.githubusercontent.com/8542534/181765060-250b7caf-6a93-4ff7-9f2a-dd5e92b9456e.png">

8. Query builder - button to edit/archive an sql snippet
<img width="485" alt="Screenshot 2022-07-29 at 15 57 31" src="https://user-images.githubusercontent.com/8542534/181765408-f57c8c9d-b908-49d6-a3ed-59993b8be364.png">

9. Notebook editor - suggested items in the expression editor
<img width="544" alt="Screenshot 2022-07-29 at 16 01 29" src="https://user-images.githubusercontent.com/8542534/181766040-95f00e05-eeed-4919-bac6-e03ef6f96232.png">

10. Item picker - (collection) list item
<img width="667" alt="Screenshot 2022-07-29 at 16 08 24" src="https://user-images.githubusercontent.com/8542534/181767147-411fafb4-bc7e-4be0-ac75-4525fff94726.png">

11. Query builder - button to download results
<img width="367" alt="Screenshot 2022-07-29 at 16 10 37" src="https://user-images.githubusercontent.com/8542534/181767474-ac935343-e19d-4e7e-873f-220bbc0a4249.png">

12. Query builder - quarter and year filter widget
<img width="1728" alt="Screenshot 2022-07-29 at 16 13 44" src="https://user-images.githubusercontent.com/8542534/181768098-2ad533cb-d27e-4ea7-9860-0424f2b83a25.png">

13. Query builder - a button to copy the link to the question in the sharing pane
<img width="668" alt="Screenshot 2022-07-29 at 16 16 37" src="https://user-images.githubusercontent.com/8542534/181768627-ca6fdd72-3bce-4055-b05b-ad144effb371.png">

14. Admin -> settings - `Learn more` link
<img width="1167" alt="Screenshot 2022-07-29 at 16 18 16" src="https://user-images.githubusercontent.com/8542534/181768841-9a194201-5a11-440d-8c60-58fc831cfbf5.png">

15. Admin -> People - menu items for custom groups
<img width="1728" alt="Screenshot 2022-07-29 at 16 19 46" src="https://user-images.githubusercontent.com/8542534/181769118-444f4896-666b-43c1-9d6a-81440156275c.png">

16. Admin -> Data Model - back button
<img width="1032" alt="Screenshot 2022-07-29 at 16 23 14" src="https://user-images.githubusercontent.com/8542534/181769738-d45ae20d-7da3-48f7-bc04-20e5f36f3763.png">

17. Admin -> Data Model - menu items for metrics and segments
<img width="1728" alt="Screenshot 2022-07-29 at 16 25 09" src="https://user-images.githubusercontent.com/8542534/181770081-e309dfd6-ec20-4a46-8eaa-0e68454bd3b8.png">
